rust/.cargo/config.toml

```diff
@@ -15,3 +15,6 @@ xclippy = [
 
 [build]
 rustflags = ["--cfg", "tokio_unstable"]
+
+[env]
+EMOJICOIN_MODULE_ADDRESS = "0xf000000d12c21b4170c7165010a4373b9ebc47bb9be9f0e48091645308a14136"
```

Comment:
I ended up leaving it in an env var as `EMOJICOIN_MODULE_ADDRESS` and instantiating the consts as lazy statics because otherwise any unit tests we make will have to instantiate the processor to know the struct/type tags, and then that becomes a pain, I'll put it in the env readme/notes